tools/file_packager.py

```diff
@@ -716,6 +716,8 @@ def was_seen(name):
         }
       };
       xhr.onerror = function(event) {
+        if (Module['setStatus']) Module['setStatus']('CORS May not be allowed on a file: URL. Use `python -m SimpleHTTPServer` to create a server at this location.');
+        throw new Error('CORS May not be allowed on a file: URL. Use `python -m SimpleHTTPServer` to create a server at this location.');
         throw new Error("NetworkError for: " + packageName);
```

Comment:
Sorry if I wasn't clear before. I was trying to say that we should not display this error twice to two different places.

Also, the throw on the second line will prevent the message on the line afterward (like 719/721).

Instead of these two lines, how about just adding some text to the throw on line 719/721?